demos/api/core_web_vitals/.gitignore

```diff
@@ -0,0 +1,107 @@
+# Tmp files
```

Comment:
This file seems overly broad. I'd suggest paring it down to only what's used.

Reply:
It was the default nodejs one from gh. pared down now.